sktime/transformations/compose/_column.py

```diff
@@ -127,13 +127,14 @@ class ColumnEnsembleTransformer(
         "fit_is_empty": False,
         "capability:unequal_length": True,
         "handles-missing-data": True,
+        "visual_block_kind": "parallel",
     }
 
     # for default get_params/set_params from _HeterogenousMetaEstimator
     # _steps_attr points to the attribute of self
     # which contains the heterogeneous set of estimators
     # this must be an iterable of (name: str, estimator, ...) tuples for the default
-    _steps_attr = "transformers"
+    _steps_attr = "_transformers"
```

Comment:
probably this is the reason?

Though, why does this not break with other trafos?

Reply:
Does the `_transformers` attribute not delegate anywhere? I think that is the problem.

Reply:
only some checks fail on this which is peculiar - did you manage to reproduce this locally?

Reply:
couldn't figure it out neither with `_transformers` nor with `_transformers_plus_remainder`. to make it work with `set_params` it would likely need a tricky setter for this property or a bigger rework of the internals. For the sake of finally closing this PR I'd go with just overriding `_sk_visual_block` by adding `remainder` there. Perhaps, we can reference this here if we were to do a bigger refactor: 
https://github.com/sktime/sktime/issues/2537

On a more general note, I think it will be difficult to avoid overriding `_sk_visual_block` if we want to add some estimator details in visualizations (e.g., displaying column names selected for a given transformer after expanding the element in case of `ColumnEnsembleTransformer`.

Reply:
ok, that makes sense. It would be appreciated if you open a new issue with the points that are not conclusively closed, so we can revisit them later.

Kudos, great PR!

Reply:
@fkiraly done, let me know if there's anything else needed before merging